app/models/pin.rb

```diff
@@ -94,7 +111,7 @@ def self.need_category
   end
 
   def self.recent
-    published.order('created_at desc')
+    published.order('updated_at desc')
```

Comment:
This is kinda a big change, but newly created content should still be at top, while anything updated will now be up too, meaning the homepage should vary a lot more.